docs/source/index.rst

```diff
@@ -49,9 +57,127 @@ a dialog into sub-dialogs and process them separately.
 Each `flow` represent a sub-dialog corresponding to the discussion of a particular topic.
 Each `flow` is also a dictionary, where the keys are the `nodes`.
 
-**Node**: Each `node` necessarily contains a `response` of the bot and
-the `condition` of `transition` to another `node`
-in this or another `flow`.
+**Node**: A `node` is the smallest unit of a dialog `flow`, and it contains the bot's response to a user's input as well as a `condition` that determines
+the `transition` to another `node`, whether it's within the current or another `flow`.
+
+ToCs
+----
+
+Context Storages
```

Comment:
It will be better to share a definition of `Context Storages` with reading people in this ReadMe